queue/management/commands/queue_ungraded_submissions.py

```diff
@@ -63,34 +70,35 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        query_exclude_params = dict(
+        grading_success_params = dict(
             lms_ack=1,
             retired=1
         )
-        query_param_dict = {}
-        query_param_list = []
+        filter_params = {}
+        exclude_params = {}
         if options['queue_names']:
-            query_param_dict['queue_name__in'] = options['queue_names'].split(',')
+            filter_params['queue_name__in'] = options['queue_names'].split(',')
         if options['submission_ids']:
-            query_param_dict['id__in'] = options['submission_ids'].split(',')
-        if options['pull_time_start']:
-            query_param_dict['pull_time__gte'] = parse_iso_8601_string(options['pull_time_start'])
-        if options['pull_time_end']:
-            query_param_dict['pull_time__lte'] = parse_iso_8601_string(options['pull_time_end'])
-            query_exclude_params['pull_time'] = None
-        else:
-            query_param_list.append(
-                Q(pull_time__lte=timezone.now() - timedelta(seconds=settings.PULLED_SUBMISSION_TIMEOUT)) |
-                Q(pull_time=None)
-            )
+            filter_params['id__in'] = options['submission_ids'].split(',')
+        if not options['include_null_pull_time']:
+            exclude_params['pull_time'] = None
+            if options['pull_time_start']:
```

Comment:
Is this nested under the null pull time conditional on purpose?

Reply:
It was on purpose, but after reading your comment I thought about it and it really isn't necessary. I guess I was thinking about users including the flag and then setting a `pull_time_start` value, which would filter out all null pull times, but if that's how someone really wants to use the command, I suppose there's no harm in letting them